integration-cli/docker_cli_links_test.go

```diff
@@ -75,22 +68,16 @@ func (s *DockerSuite) TestLinksInspectLinksStarted(c *check.C) {
 	dockerCmd(c, "run", "-d", "--name", "container2", "busybox", "top")
 	dockerCmd(c, "run", "-d", "--name", "testinspectlink", "--link", "container1:alias1", "--link", "container2:alias2", "busybox", "top")
 	links, err := inspectFieldJSON("testinspectlink", "HostConfig.Links")
-	if err != nil {
-		c.Fatal(err)
-	}
+	c.Assert(err, check.IsNil, check.Commentf("%v", err))
 
 	err = unmarshalJSON([]byte(links), &result)
-	if err != nil {
-		c.Fatal(err)
-	}
+	c.Assert(err, check.IsNil, check.Commentf("%v", err))
 
 	output := convertSliceOfStringsToMap(result)
 
 	equal := reflect.DeepEqual(output, expected)
 
-	if !equal {
-		c.Fatalf("Links %s, expected %s", result, expected)
-	}
+	c.Assert(equal, check.Equals, true, check.Commentf("Links %s, expected %s", result, expected))
```

Comment:
Maybe can use `check.DeepEquals` on `output` and `expected` directly instead of this variable and conditional.